protein/views.py

```diff
@@ -174,16 +175,19 @@ def SelectionAutocomplete(request):
             protein_source_list.append(protein_source.item)
 
         # find proteins
-        if type_of_selection!='navbar':
+        if type_of_selection!='navbar' and type_of_selection!='ligands':
             ps = Protein.objects.filter(Q(name__icontains=q) | Q(entry_name__icontains=q),
                                         species__in=(species_list),
                                         source__in=(protein_source_list)).exclude(family__slug__startswith=exclusion_slug).exclude(sequence_type__slug='consensus')[:10]
+        elif type_of_selection == 'ligands':
```

Comment:
The contains filter has a few unwanted side effects. Let's discuss them for a next update.